charts/spire/charts/tornjak-ui/values.yaml

```diff
@@ -0,0 +1,49 @@
+# Default values for Tornjak UI (Frontend).
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  # registry: gcr.io
+  # repository: spiffe-io/tornjak-fe
+  registry: ghcr.io
+  repository: spiffe/tornjak-fe
+  pullPolicy: IfNotPresent
+  # -- Overrides the image tag whose default is the chart appVersion.
+  version: "v1.0.1"
+
+imagePullSecrets: []
+nameOverride: ""
+namespaceOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # -- Specifies whether a service account should be created
+  create: true
+  # -- Annotations to add to the service account
+  annotations: {}
+  # -- The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+#labels: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 3000
+
+# tornjak - Tornjak specific configuration
+tornjakFrontend:
+  # -- URL of the Tornjak APIs (backend).
+  # The default value is used when both deployed in the same, local cluster.
```

Comment:
Since the only thing in the values is the frontend, probably don't need the tornjakFrontend config section. otherwise,at the root of the chart, it becomes:

tornjak-ui.tornjakFrontend.apiServerURL